base/staticdata.jl

```diff
@@ -0,0 +1,296 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+module StaticData
+
+using Core: CodeInstance, MethodInstance
+using Base: get_world_counter
+
+const WORLD_AGE_REVALIDATION_SENTINEL::UInt = 1 # needs to sync with staticdata.c
```

Comment:
This is the actual definition of this world, not "merely" a sentinel value, so it wouldn't actually be possible to use a different value
```suggestion
const WORLD_AGE_REVALIDATION_SENTINEL::UInt = 1
```